olive/model/utils/hfconfig_utils.py

```diff
@@ -0,0 +1,14 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from olive.model.config.hf_config import HfConfig
+from olive.model.utils.hf_utils import get_hf_model_config
+
+
+def get_loading_args(hf_config: HfConfig):
```

Comment:
Like @guotuofeng suggested in an earlier comment, I think we can just make this directly a method of `HfConfig` like it used to be before the refactor https://github.com/microsoft/Olive/blob/e337a01f0e113ed063a2839337dd4ea54ce6840e/olive/model/hf_utils.py#L221

This way, `get_model_type` can be like any other `hf_utils` helper function like:
```
def get_model_type(model_name: str, **kwargs):
    ...
```

In the conversion pass, since we already have the hf_config object, it is not complicated calling something like
```
get_model_type(hf_config.model_name, **hf_config.get_loading_args)
```

Reply:
also might be better to rename `get_loading_args -> get_from_pretrained_args` since `loading_args` is an artifact from before this renaming PR https://github.com/microsoft/Olive/pull/785

Reply:
Even if you would like to keep the current signature of `get_model_type`, I think it is better to be directly in `hf_config.py` as a helper function named like `get_model_type_from_hf_config`. We already have something similar in https://github.com/microsoft/Olive/blob/ab0af2333e17e06d2245640f1759d1b7eed955f5/olive/model/config/io_config.py#L84. 

Putting it under `model.utils.hfconfig_utils` makes the hierarchy of the utils module confusing. Ideally, I think the utils should be at a lower level than the configs or handlers. So, they won't have any knowledge of the hfconfig class.

Reply:
Done. Removed hfconfig_utils.py & moved the functions in hf_config.py